src/mono/wasm/runtime/pinvoke.c

```diff
@@ -0,0 +1,37 @@
+#include "pinvoke.h"
```

Comment:
add the license header:

```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

#include "pinvoke.h"
```